vip-application-importer/src/main/java/fr/insalyon/creatis/vip/applicationimporter/client/rpc/ApplicationImporterService.java

```diff
@@ -55,7 +55,7 @@ public static ApplicationImporterServiceAsync getInstance() {
 
     String readAndValidateBoutiquesFile(String fileLFN) throws ApplicationImporterException;
 
-    void createApplication(BoutiquesTool bt, String type, String tag, boolean isRunOnGrid, boolean overwriteVersion) throws ApplicationImporterException;
+    void createApplication(BoutiquesTool bt, String type, String tag, boolean isRunOnGrid, boolean overwriteVersion, String executionType) throws ApplicationImporterException;
```

Comment:
I prefer to use the term `fileAccessProtocol`, it would be clearer. Please use it throughout this PR.